.github/workflows/ci.yml

```
@@ -1,0 +1,83 @@
+      - name: Security check with bandit
+        run: bandit -r app/ -f json -o bandit-report.json || true
+
+      - name: Safety check for vulnerabilities
+        run: safety check --json --output safety-report.json || true
+
+      - name: Test with pytest
+        run: |
+          pytest tests/ --cov=app --cov-report=xml --cov-report=html
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)